doc/source/examples/resolved-cfd-dem/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -0,0 +1,345 @@
+
+* The ``ilu preconditioner fill`` is set to 0. This means that we don't have any preconditioner. This is the cheapest option. In this case, we are able to use this option without having to do too many ``gmres`` iterations. It requires less computational time to do a few more  ``gmres`` iterations than building the preconditioner and doing fewer ``gmres`` iterations.
+
+IB particles
+~~~~~~~~~~~~~~
+.. code-block:: text
+
+	subsection particles
+		set number of particles = 1
+		set stencil order =3
+		set refine mesh inside radius factor = 0.8
+		set refine mesh outside radius factor = 1.3
+		set initial refinement = 6
+		set fluid density =0.000960
+		set integrate motion = true
+		set assemble Navier-Stokes inside particles = false
+		set length ratio = 2
+		set particle nonlinear tolerance=1e-5
+		set alpha =1
+		subsection gravity
+		    	set Function expression =0;-981;0
+		end
+		subsection particle info 0 
+			subsection position
+			    set Function expression =5;12.75;5
+			end
+			subsection velocity
+			    set Function expression =0;0;0
+			end
+			set radius = 0.75
+			set density= 0.001120
+	    	end
+
+	end
+
+This section is where we define most of the parameters that are related to the particle.
+
+
+* The ``number of particles`` is set to one as we only want one particle.
+
+* The ``stencil order`` is set to 3 as this is the highest order we can use for this case, and it won't lead to Runge instability.
+
+* The ``refine mesh inside radius factor`` is set to 0.8. This will create a mesh refinement around the particle that avoids having hanging nodes used in the calculation and help ensure a small enough mesh around the particle.
+
+* The ``refine mesh outside radius factor`` is set to 1.3. This will create a mesh refinement around the particle that avoids having hanging nodes used in the calculation and help ensure a small enough mesh around the particle.
+
+* The ``initial refinement`` is set to 6. Here we want to have the mesh as small as possible for the first time step. To archive this, we refine every element with at least one vertex in the refinement zone around the particle 6 times before the simulation starts. This ensures that all the cell in the refinement zone around the particle is as small as possible. This number of refinement is 1 more than necessary. This is to avoid having part of the particle not properly refined as the initial mesh is big enough that some elements cut by the IB may not be properly detected at the beginning of the process. Doing one more refinement ensures that all the elements are properly refined. 
+
+* The ``fluid density`` is set to 0.000960 according to the description of the problem. Has mentioned above, this parameter is a duplication of the density parameter in the physique properties. This will be changed soon, and this parameter will be removed.
```

Comment:
```suggestion
* The ``fluid density`` is set to 0.000960 according to the description of the problem. As mentioned above, this parameter is a duplication of the density parameter in the physics properties. This will be changed soon, and this parameter will be removed.
```